lib/owners/lexer.l

```diff
@@ -5,7 +5,7 @@ macro
   TEAMNAME          @[a-zA-Z0-9\-]+\/[a-zA-Z0-9\-]+
   USERNAME          @[a-zA-Z0-9\-]+
   GLOB              [a-zA-Z0-9_\-\*\?\.]+
-  COMMENT           \# [^\"\n\"]*
+  COMMENT           \#\s[^\"\n\"]*
```

Comment:
I agree on the issue with ```\s``` and with the unnecessary (and probably undesirable) disallowing of quotes in comments.

I can make those changes and write tests to validate.

I also thought about the whitespace requirement being a bit odd. It doesn't really match most of the familiar comment syntaxes. But since I considered this a bug fix I didn't want to change the behavior of existing OWNERS files (except where it was obviously undesirable).

Switching to optional whitespace would have the advantage of simplifying the definition to just:

```COMMENT      \#[^\n]*```